viadot/sources/outlook.py

```diff
@@ -1,26 +1,27 @@
-import datetime
-from typing import Any, Dict, List, Literal
+import sys
+import pytz
+from datetime import date, datetime, timedelta
+from typing import Any, Dict, List
 
+import prefect
 import pandas as pd
 from O365 import Account
+from O365.mailbox import MailBox
 
-from viadot.config import local_config
 from viadot.exceptions import CredentialError
 from viadot.sources.base import Source
-import prefect
 
 
 class Outlook(Source):
+    utc = pytz.UTC
```

Comment:
Do you need to use the new library here? (`pytz`) Please check `datetime.timezone.utc`

Reply:
Not necessary I could switch it to `timezone.utc` because it was a problem in python 2 but now it has not sense, I used it because I used to use it.